### PR TITLE
{float,int}-to-string conversion nodes now do formatting v2

### PR DESCRIPTION
**Pull request by glima**
Changes since v1:

- @dorileo 's comments taken in account
- it now depends on #436 's HAVE_LOCALE dependency check
- log_init() changed to account for @dorileo 's fixes on logs

### CODE REVIEW

File: src/modules/flow/converter/string-format.c [hunk 1]
```diff
@@ -0,0 +1,2329 @@
...
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/* This file has code excerpts (string formatting module) extracted
+ * from cpython project (https://hg.python.org/cpython/), that comes
+ * under the PSFL license. The string formatting code was adapted here
+ * to Soletta data types. The entire text for that license is present
+ * in this directory */
```
Comment (lpereira) [inline src/modules/flow/converter/string-format.c]: IANAL but I believe you should also state the original copyright/authorship here, not only state which license this code has been based upon.

Comment (glima) [inline src/modules/flow/converter/string-format.c]: @anselmolsm ^ Can you help with this, please?

Comment (anselmolsm) [inline src/modules/flow/converter/string-format.c]: The original copyright line is missing ;)

**Comment by glima** [comment]
Thanks everyone. Copyright note added and now merging.

**Pull request closed without merging**